### PR TITLE
docs: add lagon to presets list

### PR DESCRIPTION
**Pull request by QuiiBz**
### 🔗 Linked issue

Relates to https://github.com/unjs/nitro/issues/966

### ❓ Type of change

- [x ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Add [Lagon](https://lagon.app) to the presets list, move Edgio to align it alphabetically and capitalize Heroku. We might want to wait for this PR on Nitro that improves a lot the documentation for Lagon: https://github.com/unjs/nitro/pull/1215

I'm assuming icons are handled separately, you can download the one for Lagon here: https://github.com/lagonapp/lagon/blob/main/assets/icon-white-black.png

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by florian-lefebvre** [comment]
I guess the icon, since it's not part of iconify, will require a [custom icon](https://github.com/nuxt-modules/icon#vue-component) so the svg in `docs/components/global/LogosLagon.vue`

**Comment by QuiiBz** [comment]
> I guess the icon, since it's not part of iconify, will require a [custom icon](https://github.com/nuxt-modules/icon#vue-component) so the svg in `docs/components/global/LogosLagon.vue`

Thanks, I've added an SVG for Lagon's icon (but not sure if I've done it properly, I'm not too familiar with the Nuxt ecosystem (yet))

File: docs/1.getting-started/10.deployment.md [hunk 1]
```diff
@@ -168,9 +168,10 @@ Nuxt 3 can be deployed to several cloud providers with a minimal amount of confi
 - :icon{name="ph:cloud-duotone" class="h-5 w-4 inline mb-2"} [Cleavr](https://nitro.unjs.io/deploy/providers/cleavr)
 - :icon{name="logos:cloudflare" class="h-5 w-4 inline mb-2"} [CloudFlare](https://nitro.unjs.io/deploy/providers/cloudflare)
 - :icon{name="logos:digital-ocean" class="h-5 w-4 inline mb-2"} [DigitalOcean](https://nitro.unjs.io/deploy/providers/digitalocean)
-- :icon{name="logos:firebase" class="h-5 w-4 inline mb-2"} [Firebase](https://nitro.unjs.io/deploy/providers/firebase)
-- :icon{name="logos:heroku-icon" class="h-5 w-4 inline mb-2"} [heroku](https://nitro.unjs.io/deploy/providers/heroku)
 - :icon{name="ph:cloud-duotone" class="h-5 w-4 inline mb-2"} [Edgio](https://nitro.unjs.io/deploy/providers/edgio)
+- :icon{name="logos:firebase" class="h-5 w-4 inline mb-2"} [Firebase](https://nitro.unjs.io/deploy/providers/firebase)
+- :icon{name="logos:heroku-icon" class="h-5 w-4 inline mb-2"} [Heroku](https://nitro.unjs.io/deploy/providers/heroku)
+- :icon{name="logos:lagon" class="h-5 w-4 inline mb-2 text-black dark:text-white"} [Lagon](https://nitro.unjs.io/deploy/providers/lagon)
```
Comment (florian-lefebvre) [inline docs/1.getting-started/10.deployment.md:174]:
```suggestion
- :icon{name="LagonIcon" class="h-5 w-4 inline mb-2 text-black dark:text-white"} [Lagon](https://nitro.unjs.io/deploy/providers/lagon)
```
Can you confirm @danielroe?

Comment (manniL) [reply docs/1.getting-started/10.deployment.md:174]: Not Daniel  - but yes, correct 👍

Comment (danielroe) [reply docs/1.getting-started/10.deployment.md]: this icon will need to be added to https://github.com/nuxt/nuxt.com instead. unfortunately it's still private but @manniL or I can copy it across and redeploy and hopefully we should then be good to go...

File: docs/1.getting-started/10.deployment.md [hunk 2]
```diff
@@ -168,9 +168,10 @@ Nuxt 3 can be deployed to several cloud providers with a minimal amount of confi
 - :icon{name="ph:cloud-duotone" class="h-5 w-4 inline mb-2"} [Cleavr](https://nitro.unjs.io/deploy/providers/cleavr)
 - :icon{name="logos:cloudflare" class="h-5 w-4 inline mb-2"} [CloudFlare](https://nitro.unjs.io/deploy/providers/cloudflare)
 - :icon{name="logos:digital-ocean" class="h-5 w-4 inline mb-2"} [DigitalOcean](https://nitro.unjs.io/deploy/providers/digitalocean)
-- :icon{name="logos:firebase" class="h-5 w-4 inline mb-2"} [Firebase](https://nitro.unjs.io/deploy/providers/firebase)
-- :icon{name="logos:heroku-icon" class="h-5 w-4 inline mb-2"} [heroku](https://nitro.unjs.io/deploy/providers/heroku)
 - :icon{name="ph:cloud-duotone" class="h-5 w-4 inline mb-2"} [Edgio](https://nitro.unjs.io/deploy/providers/edgio)
+- :icon{name="logos:firebase" class="h-5 w-4 inline mb-2"} [Firebase](https://nitro.unjs.io/deploy/providers/firebase)
+- :icon{name="logos:heroku-icon" class="h-5 w-4 inline mb-2"} [Heroku](https://nitro.unjs.io/deploy/providers/heroku)
+- :icon{name="LagonIcon" class="h-5 w-4 inline mb-2 text-black dark:text-white"} [Lagon](https://nitro.unjs.io/deploy/providers/lagon)
```
Comment (Atinux) [inline docs/1.getting-started/10.deployment.md:174]:
```suggestion
- :icon{name="IconLagon" class="h-5 w-4 inline mb-2 text-black dark:text-white"} [Lagon](https://nitro.unjs.io/deploy/providers/lagon)
```

**Review by atinux — APPROVED** [review]

**Review by florian-lefebvre — APPROVED** [review]

**Comment by danielroe** [comment]
Thank you @Atinux ❤️

**Review by danielroe — APPROVED** [review]

**Pull request merged**